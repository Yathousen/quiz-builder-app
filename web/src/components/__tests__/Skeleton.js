import React from 'react';
import renderer from 'react-test-renderer';
import { Skeleton } from '../';

it('Skeleton', () => {
  expect(renderer.create(<Skeleton />).toJSON()).toMatchSnapshot();
});
