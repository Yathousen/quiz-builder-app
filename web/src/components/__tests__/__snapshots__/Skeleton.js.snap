// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton 1`] = `
<div
  className="bg-white p-2 sm:p-4 sm:h-64 rounded-lg shadow-sm flex flex-col sm:flex-row gap-5 select-none "
>
  <div
    className="flex flex-col flex-1 gap-5 sm:p-2"
  >
    <div
      className="flex flex-1 flex-col gap-3"
    >
      <div
        className="bg-gray-200 w-full animate-pulse h-14 rounded-2xl"
      />
      <div
        className="bg-gray-200 w-full animate-pulse h-3 rounded-2xl"
      />
      <div
        className="bg-gray-200 w-full animate-pulse h-3 rounded-2xl"
      />
      <div
        className="bg-gray-200 w-full animate-pulse h-3 rounded-2xl"
      />
      <div
        className="bg-gray-200 w-full animate-pulse h-3 rounded-2xl"
      />
    </div>
    <div
      className="mt-auto flex gap-3"
    >
      <div
        className="bg-gray-200 w-20 h-8 animate-pulse rounded-full"
      />
      <div
        className="bg-gray-200 w-20 h-8 animate-pulse rounded-full"
      />
    </div>
  </div>
</div>
`;
